### PR TITLE
MODUIMP-18 Update documentation

### PR DESCRIPTION
**Pull request by psmagin**
## Purpose
Update module documentation and examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,41 +55,74 @@ The default <code>okapiUrl</code> is <code>http://localhost:9130</code>. The def
...
+      "customFields": {
+        "scope": "Design",
+        "specialization": [
+          "Business",
+          "Jurisprudence"
+        ]
+      },
+      "requestPreference": {
+        "holdShelf": true,
+        "delivery": false,
```
Comment (natali-zaitseva) [inline README.md:98]: delivery should be true if you "defaultDeliveryAddressTypeId" and "fulfillment"

File: README.md [hunk 2]
```diff
@@ -109,6 +142,27 @@ This should be true if only the fields present in the import should be updated,
 ### sourceType
 A prefix for the <code>externalSystemId</code> to be stored in the system. This field is useful for those organizations that has multiple sources of users. With this field the multiple sources can be separated. The source type is appended to the beginning of the <code>externalSystemId</code> with an underscore, e.g. if the user's <code>externalSystemId</code> in the import is somebody012 and the <code>sourceType</code> is test, the user's <code>externalSystemId</code> will be test_somebody012.
 
+### requestPreference
```
Comment (natali-zaitseva) [inline README.md:145]: ### requestPreference
Use this attribute to populate the user Request preference. The Request Preference contains following properties:
<code>holdShelf</code> - required field, should always be true;
<code>delivery</code> - required field, could be <code>true</code> or <code>false</code>;
<code>defaultServicePointId</code> - optional, the id of user's default service point
if <code>delivery</code> is <code>true</code> then next properties can be used
<code>fulfillment</code> - required field, can only have <code>Hold shelf</code> or <code>Delivery</code> value;
<code>defaultDeliveryAddressTypeId</code> - optional, the name of user's address type

File: ramls/examples/userdataimportCollection.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      "customFields": {
+        "scope": "Design",
+        "specialization": [
+          "Business",
+          "Jurisprudence"
+        ]
+      },
+      "requestPreference": {
+        "holdShelf": true,
+        "delivery": false,
```
Comment (natali-zaitseva) [inline ramls/examples/userdataimportCollection.json:42]: also, should be changed to true

File: README.md (hunk 2 above)
Comment (psmagin) [reply README.md:145]: Thank you for the more detailed description!
After reading I got a thought that `holdShelf` and `delivery` fields are redundant. `holdShelf` - is always true and there is no cause for user to specify it. And `delivery` can be populated based on the `fulfillment` field. Just interesting why this fields are needed.

**Review by natali-zaitseva — APPROVED** [review]

**Pull request merged**